### PR TITLE
Yapf auto-formatting configuration.

### LINKED ISSUE #19: Possible YAPF defaults
- COALESCE_BRACKETS = True
- DEDENT_CLOSING_BRACKETS = True
- SPACE_BETWEEN_ENDING_COMMA_AND_CLOSING_BRACKET = True
- SPLIT_PENALTY_AFTER_OPENING_BRACKET

cc @greut

### PR DESCRIPTION
**Pull request by greut**
- Add configuration for yapf auto-formatting. (closes #19)

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

### CODE REVIEW

File: requirements_builder/__init__.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 # Requirements-Builder is free software; you can redistribute it and/or
 # modify it under the terms of the Revised BSD License; see LICENSE
 # file for more details.
+#
```
Comment (jirikuncar) [inline requirements_builder/__init__.py]: @greut is there a way to keep the empty line between license and module docstring?

File: tests/test_cli.py [hunk 2]
```diff
@@ -58,26 +60,25 @@ def test_cli_extras():
     output = ['click==5.0.0', 'mock==1.3.0']
     with runner.isolated_filesystem():
         shutil.copytree(DATA, abspath(join(getcwd(), 'data/')))
-        result = runner.invoke(cli, [
```
Comment (jirikuncar) [inline tests/test_cli.py]: why is not PYPF more greedy and breaks the line already after first opening bracket?

File: requirements_builder/__init__.py (hunk 1 above)
Comment (greut) [inline requirements_builder/__init__.py]: I've snuck that little `#` out of despair.

File: tests/test_cli.py (hunk 2 above)
Comment (greut) [inline tests/test_cli.py]: no clues.

**Comment by greut** [comment]
@lnielsen great, thanks.

**Pull request merged**